index.js

```diff
@@ -105,6 +105,6 @@ module.exports = fp(async function (fastify, opts) {
     return results
   }
 }, {
-  fastify: '3.x',
+  fastify: '^4.x',
```

Comment:
```suggestion
  fastify: '4.x',
```

Just use the version number to avoid something like this again: https://github.com/fastify/fastify-caching/pull/103